### DISCUSSION TITLE
Line colors in Engraving preview

### DISCUSSION THREAD

**Discussion by Tribo-2dot0** [arkypita/LaserGRBL#1457] [Q&A]
![Captura de Tela (72)](https://user-images.githubusercontent.com/83982525/117717923-d07b4780-b1b1-11eb-945f-ef1275cb278a.png)
The colors of the lines that appear... Red is when the laser is on, gray is when it is transiting to the next point ... why is the initial dashed line blue? Are there any other colors?

(Google Translate)

**Comment by StuartB4** [comment]
The blue dashed line is just to show where it will start. Colour is not important.
